GenderPayGap.WebUI/Classes/Extensions/Extensions.cs

```diff
@@ -130,10 +131,10 @@ public static User FindUser(this IDataRepository repository, IPrincipal principa
 
         #region Registraion Helpers
 
-        public static async Task<bool> SendVerifyEmailAsync(this Controller controller, string emailAddress, string verifyCode)
+        public static void SendVerifyEmail(this Controller controller, string emailAddress, string verifyCode)
         {
             string verifyUrl = controller.Url.Action("VerifyEmail", "Register", new {code = verifyCode}, "https");
-            return await Emails.SendCreateAccountPendingVerificationAsync(verifyUrl, emailAddress);
+            EmailSendingService.SendCreateAccountPendingVerificationEmail(emailAddress, verifyUrl);

```

Comment:
If this method is only used by the controller, then you might as well inline the method - it's only 2 lines and it's a bit confusing how it's currently an extension method